### ISSUE TITLE
Remove prettier-related dependencies in `contracts-core`

### ISSUE THREAD

**Issue by ChiTimesChi** [synapsecns/sanguine#754]
#753 replaced prettier with `forge fmt`.

The dependencies were not removed in order to avoid merge conflicts, when the branch is eventually going to be merged into `master`.

However, once #654 is merged, the dependencies could be safely removed.
